app/views/layouts/application.html.erb

```diff
@@ -15,7 +15,7 @@
 
   <body>
     <div class="navbar-fixed">
-      <nav>
+      <nav class=<%= request.url.include?("dashboard") ? "nav-extended" : "" %>>
```

Comment:
I made some adjustments on a new pull request which I think does just this.
Please review: 'Extended navbar adjustments #60'